### PR TITLE
ci: automatically check markdown links

### PR DESCRIPTION
**Pull request by DaniBodor**
added a CI workflow to check that all links in md files are active

### CODE REVIEW

File: .mlc-config.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
```
Comment (gcroci2) [inline .mlc-config.json:1]: Is this file needed?

**Review by gcroci2 — APPROVED** [review]

**Review by gcroci2 — APPROVED** [review]

File: .mlc-config.json (hunk 1 above)
Comment (DaniBodor) [reply .mlc-config.json]: yes, but only for 1 link that is incorrectly flagged as broken. I minimized the file and moved it out of the root.
